keps/sig-storage/1885-csi-migration-azurefile/kep.yaml

```diff
@@ -33,6 +33,7 @@ latest-milestone: "v1.24"
 milestone:
   alpha: "v1.15"
   beta: "v1.21"
+  stable: "v1.26"
```

Comment:
Please update lines 25 and 30 above to reflect GA and 1.26 :)

Reply:
If I update L25 as stable, the test would fail like following:
```
time="2022-09-17T02:34:13Z" level=info msg="getting PRR approver for stable stage: the selected PRR milestone stage is not populated"
    metadata_test.go:39: 
        	Error Trace:	metadata_test.go:39
        	Error:      	"map[/home/prow/go/src/github.com/kubernetes/enhancements/keps/sig-storage/1885-csi-migration-azurefile/kep.yaml:[getting PRR approver for stable stage: the selected PRR milestone stage is not populated]]" should have 0 item(s), but has 1
        	Test:       	TestValidation
```

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/enhancements/3360/pull-enhancements-verify/1570964052291620864

Reply:
Update the [PRR approver file](https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-storage/1885.yaml) to add stage `stable`

Reply:
Still not fixed - please ping me once this is fixed and you will have SIG-level approval.